src/sampling.jl

```diff
@@ -74,12 +74,23 @@ end
 
 ### draw a pair of distinct integers in [1:n]
 
+"""
+    samplepair(n)
+
+Draw a pair of integers between 1 and `n` without replacement.
```

Comment:
Maybe you've not revised this part yet, but I think I noted that "distinct" is the key word here (and below).